frontend/src/component/PublicProfile/CustomFormManager.jsx

```diff
@@ -0,0 +1,289 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import classnames from 'classnames'
+import { translate } from 'react-i18next'
+import { IconButton } from 'tracim_frontend_lib'
+import { Checkbox } from 'tracim_frontend_lib'
+// import Form from '@rjsf/core'
+import Form from 'react-jsonschema-form-bs4'
+
+require('./CustomFormManager.styl')
+
+const DisplayTitle = props => {
+  if (!props.label) return null
+
+  return (
+    <div className='DisplayTitle'>
+      {props.label}
+    </div>
+  )
+}
+
+const DisplaySchemaPropertyString = props => {
+  return (
+    <div className='DisplaySchemaPropertyString'>
+      <span
+        className={classnames(
+          'DisplaySchemaPropertyString__label',
+          { noLabel: !props.label }
+        )}
+      >
+        {props.label}
+      </span>
+      <span className='DisplaySchemaPropertyString__value'>{props.value}</span>
+    </div>
+  )
+}
+
+const DisplaySchemaPropertyBoolean = props => {
+  return (
+    <div className='DisplaySchemaPropertyString'>
+      <span
+        className={classnames(
+          'DisplaySchemaPropertyString__label',
+          { noLabel: !props.label }
+        )}
+      >
+        {props.label}
+      </span>
+      <span className='DisplaySchemaPropertyString__value'>
+        <Checkbox
+          name={props.label}
+          checked={props.value}
+          onClickCheckbox={() => {}}
+          styleLabel={{ top: '8px', cursor: 'default' }}
+          styleCheck={{ top: '-5px' }}
+        />
+      </span>
+    </div>
+  )
+}
+
+const DisplaySchemaArray = props => {
+  if (!props.valueList) return null
+
+  if (props.valueList.every(value => typeof value === 'string')) {
+    return (
+      <DisplaySchemaPropertyString
+        label={props.label}
+        value={props.valueList.join(', ')}
+      />
+    )
+  }
+
+  return [
+    <DisplayTitle
+      label={props.label}
+      key={props.label}
+    />,
+    props.valueList.map((value, i) => {
+      const valueType = typeof value
+      if (Array.isArray(value)) return null // INFO - CH - 20210122 - I currently don't know if this case exists in Json schema spec
```

Comment:
```suggestion
      if (!value || Array.isArray(value)) return null // INFO - CH - 20210122 - I currently don't know if this case exists in Json schema spec
```
To account for value === null? (typeof null === 'object')

Reply:
case handled